lib/schemes/oauth2.js

```diff
@@ -169,7 +221,8 @@ export default class Oauth2Scheme {
           redirect_uri: this._redirectURI,
           response_type: this.options.responseType,
           audience: this.options.audience,
-          grant_type: this.options.grantType
+          grant_type: this.options.grant_type,
+          code_verifier: (this.options.code_challenge_method && this.options.code_challenge_method !== 'implicit' ? localStorage.getItem('pkce_code_verifier') : undefined)
```

Comment:
Maybe we could move that up to make it clear.

Something like this, what do you think?

```js
// Get code verifier
let codeVerifier
if (this.options.codeChallengeMethod && this.options.codeChallengeMethod !== 'implicit') {
  codeVerifier = this.$auth.$storage.getUniversal(this.name + '.code_verifier')
}
```

Reply:
And can you add code verifier to refresh token request as well please? :)

Reply:
Thank you for the review - I will address your comments in the next couple of days.

Reply:
I can break it into separate assign property - was trying to keep the one-line method going. I can add to the the refreshToken request (but I'm not sure if its needed)

Reply:
Yeah you're right. I tested and it doesn't need code verifier :)